### PR TITLE
[feature] add example for spark read data from tablestore

### PR DESCRIPTION
**Pull request by shadowinlife**
add example for spark read data from tablestore

### CODE REVIEW

File: spark/pom.xml [hunk 1]
```diff
@@ -15,6 +15,17 @@
     <packaging>jar</packaging>
     <dependencies>
         <!--for local debugging in idea, you may need comment out the dependencies' <scope>provided</scope> -->
+        <!--Spark Tablestore connector-->
+        <dependency>
+            <groupId>com.aliyun.openservices</groupId>
+            <artifactId>tablestore</artifactId>
+            <version>4.11.0</version>
+        </dependency>
```
Comment (mammothcm) [inline spark/pom.xml:24]: 会被间接依赖进去吧

File: spark/pom.xml [hunk 2]
```diff
@@ -15,6 +15,17 @@
...
+        <dependency>
+            <groupId>com.aliyun.apsaradb</groupId>
+            <artifactId>spark-tablestore</artifactId>
+            <version>4.11.0_2.4.3-1.0.4</version>
+        </dependency>
```
Comment (mammothcm) [inline spark/pom.xml:30]: provided

File: spark/pom.xml (hunk 1 above)
Comment (shadowinlife) [reply spark/pom.xml:24]: done

File: spark/pom.xml (hunk 2 above)
Comment (shadowinlife) [reply spark/pom.xml:30]: done

**Pull request merged**